sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -520,16 +520,15 @@ boolean setMaxSize(long maxStorageSizeInBytes) {
         long newMaxSize = db.setMaximumSize(maxStorageSizeInBytes);
 
         /* SQLite always use the next multiple of 4KB as maximum size. */
-        long expectedMultipleMaxSize = (long)Math.ceil((double)maxStorageSizeInBytes / (double)ALLOWED_SIZE_MULTIPLE) * ALLOWED_SIZE_MULTIPLE;
+        long expectedMultipleMaxSize = (long) Math.ceil((double) maxStorageSizeInBytes / (double) ALLOWED_SIZE_MULTIPLE) * ALLOWED_SIZE_MULTIPLE;
 
         /* So to check the resize works, we need to check new max size against the next multiple of 4KB. */
         if (newMaxSize != expectedMultipleMaxSize) {
             AppCenterLog.error(LOG_TAG, "Could not change database size to " + maxStorageSizeInBytes + " bytes, current max size is " + newMaxSize + " bytes.");
             return false;
         }
         if (maxStorageSizeInBytes != newMaxSize) {
-            AppCenterLog.warn(LOG_TAG, "Could change database size but is slightly larger than expected (next multiple of 4KB), requestedMaxSize=" +
-                    maxStorageSizeInBytes + " actualMaxSize=" + newMaxSize);
+            AppCenterLog.info(LOG_TAG, "Will attempt to change database size to " + newMaxSize + " bytes (next multiple of 4KiB).");
```

Comment:
On Android the operation is already done at this point, it's not "will". Is that different on ios?